cloudvolume/datasource/precomputed/image/__init__.py

```diff
@@ -390,26 +390,34 @@ def make_shard(self, img, bbox, mip=None, spec=None):
       else:
         raise ValueError("mip {} does not have a sharding specification.".format(mip))
 
+    bbox = Bbox.create(bbox)
```

Comment:
This change means you can't use slices with an unspecified end. Bbox.create(slices, bounds, bounded=True) calls reify_slices.

Reply:
On the other hand this is probably a useless feature here.